adafruit_httpserver/request.py

```diff
@@ -33,8 +33,35 @@ def _add_field_value(self, field_name: str, value: Union[str, bytes]) -> None:
         else:
             self._storage[field_name].append(value)
 
-    def get(self, field_name: str, default: Any = None) -> Union[str, bytes, None]:
+    @staticmethod
+    def _html_output_encode(value):
+        """Encodes unsafe HTML characters."""
+        return (
+            str(value)
+            .replace("&", "&amp;")
+            .replace("<", "&lt;")
+            .replace(">", "&gt;")
+            .replace('"', "&quot;")
+            .replace("'", "&#x27;")
+        )
+
+    @staticmethod
+    def _debug_warning_nonencoded_output():
+        """Warns about XSS risks."""
+        print(
+            "WARNING: Setting html_output_encode to False makes XSS vulnerabilities possible by "
+            "allowing access to raw untrusted values submitted by users. If this data is reflected "
+            "or shown within HTML without proper encoding it could enable Cross-Site Scripting."
+        )
+
+    def get(
+        self, field_name: str, default: Any = None, html_output_encode=True
```

Comment:
```suggestion
    def get(
        self, field_name: str, default: Any = None, *, safe=True
```

I would suggest changing name to `safe` (similar to Django) and making it a keyword only argument.

This will make it shorter and even more verbose, as one will have to write get("...", safe=False).

Name `html_output_encode` also appears in debug message.

Reply:
changed to safe and keyword only with latest commit.